tests/foreman/api/test_host.py

```diff
@@ -22,7 +22,11 @@
 from fauxfactory import gen_integer, gen_ipaddr, gen_mac, gen_string
 from nailgun import client, entities
 from requests.exceptions import HTTPError
+from six.moves import http_client
```

Comment:
I agree with you and most of times I refactored my code to put them all together. I knew [pep8](https://www.python.org/dev/peps/pep-0008/#imports) state about importing but never realized robottello doc itself mention the exactly same format. Maybe it's worth a email or issue discussion about it so everyone get aware of this. I would be happier once Pycharm follow pep 8 as well ;)